model/perf.go

```diff
@@ -170,6 +170,7 @@ func (result *PerformanceResult) Remove() (int, error) {
 type PerformanceResultInfo struct {
 	Project   string           `bson:"project,omitempty"`
 	Version   string           `bson:"version,omitempty"`
+	Order     int              `bson:"order,omitempty"`
```

Comment:
Does this need to change the hash?

Reply:
added it to the hash, though it probably doesn't need to be there because it isn't really adding to the uniqueness of a result

Reply:
On second thought, since it is not needed for uniqueness I am not going to add it to the hash. It is not worth the effort of changing the schema if it is not going to make a difference with respect to distinguishing results.